### ISSUE TITLE
Action handler for asynchronous functions

### ISSUE THREAD

**Issue by tamtam180** [commander-js/extra-typings#37]
Hi,
I found a type-warning when calling action-handler asynchronously.
In the commander.js code, the return value of the action handler is defined as `void | Promise<void>`.

commander.js
https://github.com/tj/commander.js/blob/release/10.x/typings/index.d.ts#L498

extra-typings
https://github.com/commander-js/extra-typings/blob/v10.0.2/index.d.ts#L667

```ts
//import { program } from 'commander';
import { program } from '@commander-js/extra-typings';

const actionHandler = async () => {
  // any code
};

program.command('sample').action(actionHandler); // ! type-warning
await program.parseAsync();
```

Thanks!

**Comment by shadowspawn** [comment]
Thanks for details. Popular bug this week! #33 #35

Fix coming, planning to release on Friday.

**Comment by tamtam180** [comment]
I'm sorry, I didn't see the develop branch.
Thanks for the quick response, and looking forward to the release. :D

**Comment by shadowspawn** [comment]
Released in v10.0.3
